packages/bot-web-ui/src/stores/dashboard-store.js

```diff
@@ -0,0 +1,14 @@
+import { observable, action } from 'mobx';
+
+export default class DashboardStore {
+    constructor(root_store) {
+        this.root_store = root_store;
+    }
+
+    @observable active_tab = 0;
+
+    @action.bound
+    setActiveTab(active_tab) {
+        this.active_tab = active_tab;
+    }
+}
```

Comment:
@rupato-deriv could you please convert the store .js to .ts and check all other files?